build/build-publish-solvers.xml

```diff
@@ -206,7 +206,7 @@
             <arg value="libpicosat.so" />
         </exec>
         <exec executable="strip" dir="${boolector.path}/deps/install/lib/">
-            <arg value="libminisat.so" />
+            <arg value="libminisat.so.2" />
```

Comment:
This change should be part of a different commit. Maybe split commits next time.

Reply:
Sorry. 
Will be seperate next time.